web/static/scss/_components.scss

```diff
@@ -115,6 +115,10 @@
   a.btn {
     margin: 0.7em 0 3em 0;
   }
+
+  form {
```

Comment:
OK, I took a look again. It's the way Phoenix creates a button with an action (e.g. the delete button when viewing a record), it wraps it in a form. Because of that I cannot make the buttons appear in a row, hence the `display: inline-block` on form here. 

Let me know how I should name the class and I will change it.